### PR TITLE
Fix TPA

### PR DESCRIPTION
**Pull request by haslinghuis**
Include `tpa_mode` in msp and UI

### CODE REVIEW

**Comment by ctzsnooze** [comment]
Tested! Works perfectly.
Fixes the problem where TPA values were incorrectly shown, and its great to be able to choose D or P&D options!
Thanks!

**Review by ctzsnooze — APPROVED** [review]

**Review by sugaarK — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by Asizon — CHANGES_REQUESTED** [review]

File: src/tabs/pid_tuning.html
Comment (haslinghuis) [reply src/tabs/pid_tuning.html:820]: We also have tpa old class, reverted label for the old one.

**Review by KarateBrot — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
